### DISCUSSION TITLE
Why does the print stop randomly (goes back to main menu)

### DISCUSSION THREAD

**Discussion by Moon-developer** [mriscoc/Ender3V2S1#268] [Q&A]
I recently installed this firmware on my SKR Mini E3V3 with the stock LCD. I'm able to print smaller objects. But when it runs for more than 1 hour or so it randomly beeps on the screen then the print stops and I'm back at the main menu with no way of resuming the print? 

### Details:
- Printer: Ender 3v2
- Board: SKR mini E3V3
- Stock screen
- Cura slicers:
    - print speed: 50mm/s
    - hotend: 205
    - bed: 65
    - Using M420 S1 instead of G29 to load the mesh from EEPROM

**extra**: I've tried different SD cards, speeds and temperatures.

**Comment by mriscoc** [comment]
I'm just guessing, but maybe your printer is rebooting. Many times, that behavior is caused by problems in the memory card.
I already made prints of many hours without problems at all but I have a 422 board.

**Reply by Moon-developer** [reply]
So I tried again with another SD card before posting this as a test. It almost reached 3-hour print time this time around and since I'm sitting next to it the whole time I noticed some more things about the situation that occurred:

- Stopped in one spot for a few seconds (as if it's stuck)
- Everything went quiet
- Screen never turned off once
- Screen showed a loading bar filling up over the rest of the menu (almost out of place)
- Then it beeped twice
- Fans kicked in to start cooling down the hotend since it's not trying to stay at 200 anymore.

I'm moving it now with the prepare menu to make sure it's not actually stuck and everything is moving fine too.

**Reply by mriscoc** [reply]
That was a reboot, the printer didn't get data from the card and stuck and then reboot. Clean your memory card slot, format your memory cards.

**Reply by Moon-developer** [reply]
This was a freshly formatted FAT32 card before I saved the gcode to it :thinking: could the memory card reader on the board itself be faulty maybe?

**Reply by mriscoc** [reply]
Could be, what size is your card?
